receipt_langsmith/receipt_langsmith/spark/evaluator_financial_math_viz_cache.py

```
@@ -125,14 +125,145 @@ def _build_involved_word(
+        equations.append(
+            {
+                "issue_type": "SUBTOTAL_BALANCED",
+                "description": "Subtotal matches sum of line items",
+                "expected_value": None,
+                "actual_value": None,
+                "difference": 0.0,
+                "involved_words": involved,
+            }
+        )
+
+    # Line Item equations: per line_id, QUANTITY × UNIT_PRICE = LINE_TOTAL
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)